doc/sphinx/practical-tools/utilities.rst

```diff
@@ -124,22 +124,28 @@ avoid putting your code into subdirectories such as "src" unless you want
 that to be part of the :term:`logical path` in Coq.
 
 For example, here's a minimal _CoqProject file for the `MyProject` package
-(the logical name of the package), which has two script files `File1.v`
-and `SubDir/File2.v` whose logical paths
-are `MyProject.File1` and `MyProject.SubDir.File2`.  `_CoqProject` should be
-in the top-level directory of your project.  We recommend naming that directory
-`MyProject`, the same as the logical name::
+(the logical name of the package), which includes all the ``.v`` files (and
+other file types) in the current directory and its subdirectories:
 
   -R . MyPackage
+  .
+
+You can list multiple directories if you wish.
+
+In addition, you can list individual files, for example for the two script files
+`File1.v` and `SubDir/File2.v` whose logical paths are `MyProject.File1` and
+`MyProject.SubDir.File2`.  We recommend naming that directory `MyProject`, the
+same as the logical name::
+
+  -R . MyProject
```

Comment:
Beware the new text mixes `-R . MyPackage` and `-R . MyProject`. That might be because the old text was already incorrect — it uses `-R . MyPackage` to configure a "logical name" `MyProject`, which is just wrong.

At a first look, there isn't a real distinction, so the simplest fix is to unify them.

Reply:
Thanks, got it.  They are all `MyPackage` now.